src/core/meshcop/dataset.hpp

```diff
@@ -672,6 +672,20 @@ class Dataset
      */
     const Tlv *GetTlvsEnd(void) const { return reinterpret_cast<const Tlv *>(mTlvs + mLength); }
 
+    /**
+     * Determines whether this Dataset is a subset of another Dataset.
+     *
+     * The Dataset is considered a subset if all of its TLVs, excluding Active/Pending Timestamp and Delay Timer TLVs,
+     * are present in the @p aOther Dataset and the TLV values match exactly.
+     *
+     * @param[in] aOther   The other Dataset to check against.
+     *
+     * @retval TRUE   The current dataset is a subset of @p aOther.
+     * @retval FALSE  The current Dataset is not a subset of @p aOther.
```

Comment:
```suggestion
     * @retval FALSE  The current dataset is not a subset of @p aOther.
```
Consistent with the previous line.

Reply:
Thanks.